components/nexusoperations/events.go

```diff
@@ -51,7 +51,7 @@ func (d ScheduledEventDefinition) Apply(root *hsm.Node, event *historypb.History
 	return err
 }
 
-func (d ScheduledEventDefinition) CherryPick(root *hsm.Node, event *historypb.HistoryEvent) error {
+func (d ScheduledEventDefinition) CherryPick(root *hsm.Node, event *historypb.HistoryEvent, shouldExcludeNexusEvent bool) error {
```

Comment:
Back to the discussion we had, we should give more control to users to only cherry pick started or terminal events. But this is fine for the time being.

Reply:
Agree. Also, the whole exclude events option is a bit confusing. We need to do some cleanup of the reset options.